app/controllers/state_file/questions/az_extension_payments_controller.rb

```diff
@@ -0,0 +1,14 @@
+module StateFile
+  module Questions
+    class AzExtensionPaymentsController < QuestionsController
+      def self.show?(intake)
+        Flipper.enabled?(:extension_period) && after_april_16_arizona_time?
+      end
+
+      def self.after_april_16_arizona_time?
+        arizona_time = Time.use_zone("Arizona") { Time.zone.now }
+        arizona_time >= Time.use_zone("Arizona") { Time.zone.parse("2025-04-16 00:00:00") }
```

Comment:
I think what's above probably works fine? But wanted to share options since we have something called `app_time`. Also for parsing a date in the timezone, I took the example from `application.rb`. 

```suggestion
        arizona_time = app_time.in_time_zone("Arizona")
        arizona_time >= Time.find_zone('Arizona').parse("2025-04-16 00:00:00")
```
Also note: [there's some refactor happening right now that is defining a `payment_deadline` for each state](https://github.com/codeforamerica/vita-min/pull/5726/files), which I think would be great to utilize here in some way

Reply:
in fact, is there a way to structure this so each state could define a timezone & it could be re-used for the next state? I think we should name the method more generically too. It can happen in this PR or the next.

Reply:
actually, we have to use `app_time` here b/c we have to be able to use session toggles to change current time for testing ty @mpidcock !

Reply:
(I don't think you need to wait for that PR to get merged since it wouldn't rely on that code, just follow the same pattern)

Reply:
that sounds nice!

Reply:
@mpidcock don't I need the payment_deadline_date and before_payment_deadline methods from state service from that pr?